lnwallet/channel.go

```diff
@@ -1865,6 +1868,10 @@ func (lc *LightningChannel) restoreStateLogs(
 			incomingRemoteAddHeights[r.HtlcIndex] =
 				pendingRemoteCommit.height
 		}
+
+		for _, r := range pendingRemoteCommit.outgoingHTLCs {
```

Comment:
I think we need a few comments in the code here to explain what this is doing

Reply:
Done